thirdeye-server/src/main/java/ai/startree/thirdeye/resources/ApiResource.java

```diff
@@ -158,4 +161,9 @@ public TaskResource getTaskResource() {
   public UiResource getUIResource() {
     return uiResource;
   }
+
+  @Path("namespace-configuration")
```

Comment:
NamespaceResource is using the name `workspaces` so let's also use `workspace` instead of `namespace` here
https://github.com/startreedata/thirdeye/blob/61b2be6ded9ae74b95c3cfc3f63dc6e7b1e9c619/thirdeye-server/src/main/java/ai/startree/thirdeye/resources/ApiResource.java#L122

Reply:
done